### PR TITLE
SLING-10651: add java.time.Instant to RuntimeObjectModel

### PR DESCRIPTION
**Pull request by buuhuu**

### CODE REVIEW

File: src/main/java/org/apache/sling/scripting/sightly/render/RuntimeObjectModel.java [hunk 1]
```diff
@@ -95,6 +96,14 @@
      */
     Date toDate(Object object);
 
+    /**
+     * Convert the given object to a {@link Instant} object
```
Comment (raducotescu) [inline src/main/java/org/apache/sling/scripting/sightly/render/RuntimeObjectModel.java]: `s/to a/to an/` :)

**Review by raducotescu — APPROVED** [review]
Thank you for the PR, Dirk! LGTM, with the exception of that tiny typo.

**Comment by kwin** [comment]
I think we need now a table of which types are supported in which use providers. I added https://issues.apache.org/jira/browse/SLING-10652 to track that.

File: src/main/java/org/apache/sling/scripting/sightly/render/RuntimeObjectModel.java (hunk 1 above)
Comment (buuhuu) [reply src/main/java/org/apache/sling/scripting/sightly/render/RuntimeObjectModel.java]: fixed the c&p typo in https://github.com/apache/sling-org-apache-sling-scripting-sightly-runtime/pull/7/commits/026497d9d0d1eba7ca4a95c629d532331d68f0b3

**Pull request merged**